src/clj/rems/email/core.clj

```diff
@@ -53,6 +54,20 @@
                      (remove nil?))]
     (enqueue-email! email)))
 
+(defn- render-invitation-template [invitation]
+  (let [lang (:default-language env)] ; we don't know the preferred languages here since there is no user
+    (cond (:invitation/workflow invitation)
+          (let [workflow (workflow/get-workflow (get-in invitation [:invitation/workflow :workflow/id]))]
+            (assert workflow "Can't send invitation, missing workflow")
+            (template/workflow-handler-invitation-email lang invitation workflow)))))
```

Comment:
could this be a `when` instead of `cond`? as there is only single condition currently

Reply:
I think this was intentional since @Macroz has explicitly said in the comment an invitation could be for smth else, but good point!

Reply:
Yeah, this is just preparing for the future invitation types. There are a few of these scattered around.